tests/test_fsevents.py

```diff
@@ -247,6 +247,58 @@ def test_converting_cfstring_to_pyunicode():
         emitter.stop()
 
 
+def test_recursive_check_accepts_relative_paths():
```

Comment:
Why is it called "recursive"?
The problem is with non-recursive and the test correctly selects non-recursive watch.